src/Dockerfile

```diff
@@ -83,11 +83,12 @@ RUN if   [ "$TARGETPLATFORM" = "linux/amd64" ];    then FTLARCH=amd64; \
     && echo "Arch: ${TARGETPLATFORM}, FTLARCH: ${FTLARCH}" \
     && curl -sSL "https://ftl.pi-hole.net/${FTL_BRANCH}/pihole-FTL-${FTLARCH}" -o /usr/bin/pihole-FTL \
     && chmod +x /usr/bin/pihole-FTL \
-    && readelf -h /usr/bin/pihole-FTL || cat /usr/bin/pihole-FTL
+    && readelf -h /usr/bin/pihole-FTL || (echo "Error with downloaded FTL binary" && exit 1)
```

Comment:
Another suggestion:

```suggestion 
    && readelf -h /usr/bin/pihole-FTL 2>&1 > /dev/null || exit 1
```

Reply:
On error, this will show:
`readelf: Error: Not an ELF file - it has the wrong magic bytes at the start`

Reply:
That error message is still shown, I personally think it's more useful to also have "Error with downloaded FTL binary" message"

```
0.450 readelf: Error: Not an ELF file - it has the wrong magic bytes at the start
0.450 Error with downloaded FTL binary
```